### PR TITLE
JHy python license added

### PR DESCRIPTION
**Pull request by x3mSpeedy**
Added script which will add/remove/replace license
 - supports placeholders such as filename filepath
 - if git location is given, supports branch, last_change and more variables

### CODE REVIEW

File: bin/python/license_manager_script.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    parser.add_option("-l", "--license", dest="license_path", metavar="FILENAME", default=None,
+                      help="Absolute or relative path to file containing new license,"
+                           " if not specified, license will be only removed")
+    parser.add_option("-f", "--file", dest="files", metavar="FILENAME", default=[], action="append",
+                      help="Absolute or relative path to file which will be processed")
+    parser.add_option("-d", "--dir", dest="dirs", metavar="DIRNAME", default=[], action="append",
+                      help="Absolute or relative path to dir which will be recursively processed")
+
+    parser.add_option("-g", "--git", dest="git", metavar="DIRNAME", default=None,
+                      help="Location of git repository (will add additional information to license)")
```
Comment (jbrezmorf) [inline bin/python/license_manager_script.py]: Do no understand to this help.

File: bin/python/license_manager_script.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    parser.add_option("-s", "--start", dest="license_start", metavar="START_SEQ", default='/*!',
```
Comment (jbrezmorf) [inline bin/python/license_manager_script.py]: What is exactly meaning of this? Is it used to search for the licence or for wrapping the text provided in file.
Should provide some help how the script works in general. In particular how to remove/replace existing licence texts that
can have some variants.  Also sometime the licence comment is followed by file comment. Need some discussion how to deal with this now and in future.

File: bin/python/license_manager_script.py (hunk 1 above)
Comment (x3mSpeedy) [inline bin/python/license_manager_script.py]: license file supports placeholders, when script is called with valid git root ```--git git-root-dir-path```, placeholders offer some git related info:
```python
"""
This is license for file '{filename}' in branch {branch}, last edited by {last_author}
License goes here
"""
```
will produce something like:
```
This is license for file 'main.cc' in branch master, last edited by Jan Brezina
License goes here
```

File: bin/python/license_manager_script.py (hunk 2 above)
Comment (x3mSpeedy) [inline bin/python/license_manager_script.py]: @jbrezmorf I added [README.md](https://github.com/x3mSpeedy/Flow123d-python-utils/blob/master/src/README.md) to my repo describing how to use this script. I added light version of this documentation to script itself in 73a63e95d732dc545b40124e0d0abc86bf820ea5

**Comment by jbrezmorf** [comment]
Final pull request  comes latter.

**Pull request closed without merging**